### PR TITLE
docs: add continous profiling docs

### PR DESCRIPTION
**Pull request by JonasBa**
## DESCRIBE YOUR PR
*Tell us what you're changing and why. If your PR **resolves an issue**, please link it so it closes automatically.*

## IS YOUR CHANGE URGENT?  

Help us prioritize incoming PRs by letting us know when the change needs to go live.
- [ ] Urgent deadline (GA date, etc.): 
- [ ] Other deadline: 
- [ ] None: Not urgent, can wait up to 1 week+

## SLA

- Teamwork makes the dream work, so please add a reviewer to your PRs.
- Please give the docs team up to 1 week to review your PR unless you've added an urgent due date to it.
Thanks in advance for your help!

## PRE-MERGE CHECKLIST

*Make sure you've checked the following before merging your changes:*

- [ ] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs (subject matter experts)
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## LEGAL BOILERPLATE

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

## EXTRA RESOURCES

- [Sentry Docs contributor guide](https://docs.sentry.io/contributing/)

### CODE REVIEW

**Review by lizokm — APPROVED** [review]
Thanks for adding this! Made some suggestions, but it's looking good.

File: docs/product/explore/profiling/index.mdx [hunk 1]
```diff
@@ -51,16 +51,34 @@ From this view, you can also click "View Profile" to zoom in on the flame graph.
...
+The "Slowest Functions" widget shows you a list of the functions that took the most time in your application. It lists the transactions the function was called from. Clicking on a transaction will take you to an example profile.
 
 Using the Profiling page is typically an advanced workflow, enabling you to directly select transactions of interest and examine detailed profiling data.
 
+### Flamegraph tab
+
+The flamegraph tab shows an aggregate flamegraph based off the query you selected. This tab is a great place to visualize a flamegraph across transaction boundaries and better understand how the system as a whole might be bottlenecked. The aggregate flamegraph is a more advanced workflow that serves to users which have setup continuous profiling and are interested in how their entire system performs.
```
[the remark this answers is not in the document]
Comment (Zylphrex) [reply docs/product/explore/profiling/index.mdx:71]: There is no distinction between an aggregate flamegraph vs flamegraph. Saying aggregate is to be explicit as people often confuse the notion of a flamegraph (aggregate) vs flamechart (singular).

File: docs/product/explore/profiling/index.mdx [hunk 2]
```diff
@@ -51,15 +51,29 @@ From this view, you can also click "View Profile" to zoom in on the flame graph.
 
 ## Profiling Page
 
-The **Profiling** page shows a list of transactions in descending order of execution time for your selected project(s).
+On the [**Profiling**](https://sentry.io/orgredirect/organizations/:orgslug/profiling/) page, you'll find two tabs:  "transactions" and "flamegraph". The transactions tab lists transactions in descending order of execution-time for your selected project(s), while the flamegraph tab shows an aggregate flamegraph that spans across transactions. Depending on how profiling has been set up, you might be interested more in one than the other.
 
-![Profiling page](./img/profiling-page-view.png)
+Using the Profiling page is typically an advanced workflow, where you to directly select transactions of interest and examine detailed profiling data.
```
Comment (lizokm) [inline docs/product/explore/profiling/index.mdx:56]:
```suggestion
Using the Profiling page is typically an advanced workflow where you can select transactions of interest directly and closely examine profiling data.
```

File: docs/product/explore/profiling/index.mdx [hunk 3]
```diff
@@ -51,15 +51,29 @@ From this view, you can also click "View Profile" to zoom in on the flame graph.
...
-The "Profiles by Percentiles" timeline chart can be used to filter the transaction data by highlighting the desired time period. Below that, there are three widgets to choose from.
+### Transactions Tab
 
-The "Slowest Functions" widget shows you a list of the functions that took the most time in your application. It lists the transactions the function was called from. Clicking on a transaction will take you to an example profile. The "Most Regressed Functions" and "Most Improved Functions" widgets show you a list of the functions that have gotten slower/faster over time. Clicking on the timings will take you to an example from either before/after the detected regression/improvement.
+This page shows a list of transactions in descending order of execution-time for your selected project(s).
 
-Using the Profiling page is typically an advanced workflow, enabling you to directly select transactions of interest and examine detailed profiling data.
+![Transactions View](./img/profiling-page-transactions-view.png)
+
+The "Profiles by Percentiles" timeline chart can be used to filter the transaction data by highlighting the desired time period.
```
Comment (lizokm) [inline docs/product/explore/profiling/index.mdx:64]:
```suggestion
The "Profiles by Percentiles" timeline chart can be used to filter transaction data by highlighting a desired time period.
```

File: docs/product/explore/profiling/index.mdx [hunk 4]
```diff
@@ -71,18 +85,14 @@ While the **Profiling** page can be used to view information about the profiles
 
 This information allows you to dive deeper into what your code was executing during this transaction and may get you closer to the root of the problem.
 
-The top of the page has a percentile chart with profile durations and a search bar. It helps you visually identify possible performance regressions or changes. You can isolate the date-time ranges you want to investigate by clicking and dragging it. If you want to narrow down your investigation to profiles executed under a specific environment, use the search bar filters.
+If you want to narrow down your investigation to profiles executed under a specific environment, use the search bar filters.
```
Comment (lizokm) [inline docs/product/explore/profiling/index.mdx:88]:
```suggestion
If you want to narrow down your investigation to profiles executed in a specific environment, use the search bar filters.
```

File: docs/product/explore/profiling/index.mdx [hunk 5]
```diff
@@ -71,18 +85,14 @@ While the **Profiling** page can be used to view information about the profiles
...
 
-The bottom right side of the page has aggregated information about function regressions detected during your selected time range plus, a list of the slowest functions. There's also a list of general information about the profile including:
+The right right side of the page has a list of the slowest functions. There's also a list of general information about the profile including:
```
Comment (lizokm) [inline docs/product/explore/profiling/index.mdx:90]:
```suggestion
The right side of the page has a list of the slowest functions. There's also a list of general information about the profile including:
```

**Review by lizokm — APPROVED** [review]
Just a few more tweaks. Looking good!

**Pull request merged**